mysql-test/main/max_used_connections_time.test

```diff
@@ -0,0 +1,35 @@
+-- source include/not_embedded.inc
```

Comment:
I would move this test to `main/status.test` where `max_used_connections` tests are.